### PR TITLE
Serenytics

### PR DESCRIPTION
**Pull request by pfd**
## What do these changes do?

Adds the Serenytics cloud destination doc

Docs preview: https://deploy-preview-941--rudderstack-docs-staging.netlify.app/destinations/streaming-destinations/serenytics/

## Type of documentation

- [x] New documentation
- [ ] Documentation update
- [ ] Hotfix

### CODE REVIEW

File: docs/destinations/streaming-destinations/serenytics.mdx [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+rudderanalytics.identify("1hKOmRA4el9Zt1WSfVJIVo4GRlm", {
+  likes_movies: true,
+  favorite_color: "purple",
+  age: 13
+});
+```
+
+**Event parameter mapping**:
+
+| RudderStack property | Serenytics property | Required? |
```
Comment (sauravlal-rudderstack) [inline docs/destinations/streaming-destinations/serenytics.mdx:100]: some typo for all fields `Required?`

File: docs/destinations/streaming-destinations/serenytics.mdx [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+| originalTimestamp | original_timestamp | No |
+| timestamp | timestamp | No |
+
+### Track
+
+The <Link to="/event-spec/standard-events/track">`track`</Link> call lets you capture user events along with the properties associated with them.
+
+**Event parameter mapping**:
+
+| RudderStack property | Serenytics property | Required? |
```
Comment (sauravlal-rudderstack) [inline docs/destinations/streaming-destinations/serenytics.mdx:120]: same

**Review by ishachopra0401 — APPROVED** [review]

File: docs/destinations/streaming-destinations/serenytics.mdx [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+
+| RudderStack property | Serenytics property | 
+| --- | --- | 
+| `userId` <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span>| `user_id` | 
+| `previousId` <br/>`anonymousId` | `previous_id` | 
+| `context.ip` <br/>`request_ip` | `context_ip` | 
+| `receivedAt` | `received_at` | 
+| `sentAt` | `sent_at` | 
+| `originalTimestamp` | `original_timestamp` |
+| `timestamp` | `timestamp` |
```
Comment (sauravlal-rudderstack) [inline docs/destinations/streaming-destinations/serenytics.mdx:155]: The mapping of group call is not correct please refer the docs.

**Comment by sauravlal-rudderstack** [comment]
My suggestion: An image of the storage URL is helpful for users to understand more.

**Pull request merged**